libbeat/metric/system/cgroup/reader.go

```diff
@@ -157,7 +156,6 @@ func (r *Reader) CgroupsVersion(pid int) (CgroupsVersion, error) {
 		// V1 and V2 controllers on a cgroup. If the V2 controller has no actual controllers associated with it,
 		// We revert to V1. If it does, report V2. In the future, we may want to "combine" V2 and V1 metrics somehow.
 		if len(controllers) > 0 {
-			fmt.Printf("fetching V2 controller: %#v for pid %d\n", controllers, pid)
```

Comment:
That looked like a print debug that was forgotten. Is it ok to remove it @fearful-symmetry ?

Reply:
That's the only change on this file, but the linter runs on all files that had any changes.

Reply:
This file is going to be removed in https://github.com/elastic/beats/pull/31615

Please move this fix to the `elastic-agent-system-metrics` repo.